### PR TITLE
feat(hpc-vmware-managed-vcd): fix data refresh on vcd/vdc updates

### PR DESCRIPTION
**Pull request by pauldkn**
ref: MANAGER-15400

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hpc-vcd-beta`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | fix/MANAGER-15400
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/UpdateDetailModalHandler.tsx [hunk 1]
```diff
@@ -37,6 +43,12 @@ export const UpdateDetailModalHandler = ({
         t(`managed_vcd_dashboard_edit_${detailName}_modal_success`),
         true,
       );
+      queryClient.invalidateQueries({
+        queryKey: getVcdOrganizationQueryKey(id),
+      });
+      queryClient.invalidateQueries({
+        queryKey: [icebergListingQueryKey, organizationListingContainerId],
+      });
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/UpdateDetailModalHandler.tsx:51]: Why 2 calls and not only one ?

File: packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/organizations/Organizations.page.tsx [hunk 2]
```diff
@@ -17,6 +17,8 @@ import { urls } from '@/routes/routes.constant';
 import IVcdOrganization from '@/types/vcd-organization.interface';
 import { VCD_ORGANIZATION_ROUTE } from '@/data/api/hpc-vmware-managed-vcd.constants';
 
+export const organizationListingContainerId = 'organizations';
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/organizations/Organizations.page.tsx:20]: if it's a constant, put in Organizations.constant.ts rather

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/datagrid/container/DatagridContainer.component.tsx [hunk 3]
```diff
@@ -20,6 +20,8 @@ export type TDatagridContainerProps = {
   orderButton?: React.JSX.Element;
 };
 
+export const icebergListingQueryKey = 'servicesListingIceberg';
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/datagrid/container/DatagridContainer.component.tsx:23]: To follow the guidelines, use a constant file rather

File: packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/edit/EditVdcDescription.page.tsx [hunk 4]
```diff
@@ -23,6 +30,13 @@ export default function EditVdcDescription() {
         t('managed_vcd_dashboard_edit_description_modal_success'),
         true,
       );
+      queryClient.invalidateQueries({
+        queryKey: getVcdDatacentreQueryKey(id, vdcId),
+      });
+      queryClient.invalidateQueries({
+        queryKey: [icebergListingQueryKey, getVdcListingContainerId(id)],
+        exact: true,
+      });
```
Comment (tibs245) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/edit/EditVdcDescription.page.tsx:39]: It's better to include it on hook directly ?

**Review by chipp972 — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by tibs245 — APPROVED** [review]

**Pull request merged**